sympy/matrices/expressions/matadd.py

```diff
@@ -33,12 +32,10 @@ class MatAdd(MatrixExpr, Add):
 
     def __new__(cls, *args, **kwargs):
         args = list(map(sympify, args))
-        check = kwargs.get('check', False)
+        check = kwargs.get('check', True)
```

Comment:
the purpose of `check` is to allow `Dummy` symbols in matrix expressions (ignore `.shape` check for dummies).